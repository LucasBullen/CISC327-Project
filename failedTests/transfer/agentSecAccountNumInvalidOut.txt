1,6c1,6
< Select mode: atm or agent.
< Logged into agent mode.
< Enter first account number.
< Enter second account number.
< Incorrect account number format (8 digits, no leading zeros). Transfer cancelled.
< Logged out.
---
> Select mode: atm or agent.
> Logged into agent mode.
> Enter number of account to deposit into.
> Enter number of account to transfer to.
> Account number does not exist. Transfer cancelled.
> Logged out.
