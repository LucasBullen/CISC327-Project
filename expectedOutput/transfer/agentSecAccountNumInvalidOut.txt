Select mode: atm or agent.
Logged into agent mode.
Enter number of account to deposit into.
Enter number of account to transfer to.
Account number does not exist. Transfer cancelled.
Logged out.
